src/overlays/actors/ovl_Bg_Kin2_Bombwall/z_bg_kin2_bombwall.c

```diff
@@ -3,20 +3,26 @@
  * Overlay: ovl_Bg_Kin2_Bombwall
  * Description: Ocean Spider House - Bombable Wall
  */
-
 #include "z_bg_kin2_bombwall.h"
+#include "objects/object_kin2_obj/object_kin2_obj.h"
 
 #define FLAGS (ACTOR_FLAG_10 | ACTOR_FLAG_10000000)
 
-#define THIS ((BgKin2Bombwall*)thisx)
+#define THIS ((BgKin2Bombwall*)(thisx))
```

Comment:
```suggestion
#define THIS ((BgKin2Bombwall*)thisx)
```
I'd say don't worry about messing with the `THIS` macro.

Reply:
In this case it's not an argument, it's just a fixed copypaste: thisx is always the same thing, so it's fine.

Reply:
Perfect!